Susi/Helpers/ControllerConstants.swift

```diff
@@ -257,6 +257,11 @@ class ControllerConstants {
         static let roomsIcon = UIImage(named: "room-icon")
     }
     
+    struct BannerNotification {
+        static let dangerTitle =  "No Internet Connection! Connect to Internet"
```

Comment:
Change it to `No internet connection! Please try connecting`

Reply:
changes done , please review